front/src/config/i18n/fr.json

```diff
@@ -176,7 +176,9 @@
       "temperature-in-room": "Température de la pièce",
       "user-presence": "Utilisateurs présent",
       "camera": "Caméra",
-      "devices-in-room": "Appareils de la pièce"
+      "devices-in-room": "Appareils de la pièce",
+      "chart-one-feature": "Graphique 1 feature",
+      "chart-multi-feature": "Graphique multi feature"
```

Comment:
This is not french

Reply:
If I replace feature by 'capteur' is ok for you
any other idea for translate feature in french ?